onnxruntime/contrib_ops/rocm/bert/skip_layer_norm_impl.cu

```diff
@@ -46,71 +46,15 @@ Status LaunchSkipLayerNormKernel(
   // this must be true because element_count is the total size of the tensor
   assert(element_count % ld == 0);
 
+  SkipLayerNormParams<T> op_params(stream, output, input, skip, gamma, beta, bias, epsilon, ld, element_count);
+
   if (tuning) {
```

Comment:
I think this part of the code can now be written as:

static SkipLayerNormTunableOp<T> op;
if (tuning) {
  op.EnableTuning();
}
return op(&op_params);